### ISSUE TITLE
Cannot find module 'sql.js/module'

### ISSUE THREAD

**Issue by EarMaster** [citycide/trilogy#109]
## description

I try to use trilogy in a TypeScript project I'm currently working on. As soon as I use ``import { connect } from 'trilogy'`` somewhere I am presented with the error below by ``tsc``:

I planned on using ``sqlite3`` as backend therefore I only used ``npm i sqlite3``, but the error appears wether I am defining the client to be ``sqlite3`` or ``sql.js``. I also tried to ``npm i sql.js``, but the error persists.

### steps to reproduce

1. Create a new project
2. Run ``npm install typescript @types/node``
3. Run ``npm install trilogy sqlite3``
5. Create ``index.ts`` with the source code below
6. Run ``npx tsc index.ts``

### source

```typescript
import { connect } from 'trilogy'

const db = connect(':memory:')
```

### error

```
node_modules/trilogy/dist/index.d.ts:4:23 - error TS2307: Cannot find module 'sql.js/module' or its corresponding type declarations.

4 import { SqlJs } from 'sql.js/module';
```

## environment
I am using Windows 10 Pro 64 Bit, Node v14.15.3 and npm 6.14.9.

**Comment by citycide** [comment]
Yeah, confirmed this is an issue. I'm not sure what's changed since this code was originally put in, but it should be a relatively easy fix to adjust those imports.

A quick glance through the history of sql.js made it seem like that entry point hasn't been a thing for quite a while ¯\\\_(ツ)_/¯

**Comment by EarMaster** [comment]
Thank you!

**Comment by citycide** [comment]
Sure thing, this is released in `v2.0.4`, please give it a shot!
